modules/aleph/src/Aleph/Handler/AlephPatronHandler.php

```diff
@@ -92,6 +92,16 @@ public function getPatron() {
   }
 
   /**
+   * Get patron debts.
+   *
+   * @return \SimpleXMLElement
+   *    The response from Aleph with debts + material info.
+   */
+  public function getDebts() {
+    return $this->client->getDebts($this->getPatron());
```

Comment:
Avoid returning SimpleXML instances from the handler. The Handlers job is to interact with the client, and do whatever and handle all of the communication with it including going from XML to entities. So take the same approach as with authentication:  Debt entity and return that instead. Create the object with a factory just like you do with AlephUser.